### PR TITLE
Save reports

### PR DESCRIPTION
**Pull request by hoslack**
Made an action that saves reports to the database 
relates #19

### CODE REVIEW

**Review by MynahMarie — APPROVED** [review]

**Review by Karyum — CHANGES_REQUESTED** [review]
In both actions ( emailSending and createReport ) you are still making the request in case it fails what will happen?? would you just say the email has been sent while it errored? would you still say to the user that the report was created while it errored too ?

**Comment by Karyum** [comment]
what we need here is error handling i know this is big but we just cant let this pass to production, cause you are just returning the action object whenever if failed or succeeded.

File: client/package.json [hunk 1]
```diff
@@ -2,7 +2,13 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
+  "proxy": {
+    "/api/*": {
+      "target": "http://localhost:5000"
+    }
+  },
   "dependencies": {
+    "axios": "^0.17.1",
```
Comment (Karyum) [inline client/package.json:11]: also when you work on something make sure you have the latest master, cause on master you are using `superagent` and not `axios` so let's be consistent with what we are using

Comment (Karyum) [reply client/package.json:11]: https://github.com/FACN3/easyRep/blob/master/client/package.json#L16 this is your master package.json

Comment (Karyum) [reply client/package.json:11]: either get rid of superagent or axios

Comment (hoslack) [reply client/package.json:11]: Error handling has been done,

**Review by MarlenAw — APPROVED** [review]

File: client/package.json (hunk 1 above)
Comment (Karyum) [reply client/package.json:11]: where ?

Comment (Karyum) [reply client/package.json:11]: whats with axios  and the requests that youre making

Comment (hoslack) [reply client/package.json:11]: Error handling has been done. I have also removed superagent
@Karyum

Comment (Karyum) [reply client/package.json]: @hoslack you are also using `superagent` https://github.com/FACN3/easyRep/blob/d0e4795afcc133eabef1f18a487d76717e5f663b/client/src/containers/UploadForm.js here, you don't just need to remove it, you also need to refactor this function to use axios

**Review by Karyum — APPROVED** [review]

**Pull request merged**